lib/tjbot.js

```diff
@@ -285,20 +288,28 @@ TJBot.prototype._setupServo = function(pin) {
  * Configure the microphone for speech recognition.
  */
 TJBot.prototype._setupMicrophone = function() {
-    if (this.configuration.verboseLogging) {
+    if (this.config.verboseLogging) {
         console.info("> TJBot initializing microphone");
     }
 
     this._mic = mic({
         'rate': '44100',
         'channels': '2',
         'debug': false,
+        'device': 'plughw:0,0',
```

Comment:
@victordibia my RPi is comming so I didn't test that there. If you could test...
I think 0,0 (card 0 and device 0) is the default values, but is a good idea to test.
Command to check card / device: arecord -l